ccdproc/ccddata.py

```diff
@@ -308,6 +308,12 @@ def to_hdu(self, hdu_mask='MASK', hdu_uncertainty='UNCERT',
             Default is ``'MASK'`` for mask, ``'UNCERT'`` for uncertainty and
             ``None`` for flags.
 
+        wcs_relax : bool
```

Comment:
I think the description should also link to http://docs.astropy.org/en/stable/api/astropy.wcs.WCS.html#astropy.wcs.WCS.to_header.